css/style.css

```diff
@@ -352,3 +352,69 @@ button:hover {
 .css {
   color: blue;
 }
+
+/* <<<<<Footer>>>>> */
+
+/* footer {
+  background-color: #f5f5f5;
+} */
```

Comment:
- [ ] I suggest removing this unused code because it will make your project look messy. [HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👌

![image](https://user-images.githubusercontent.com/45231276/177459985-677e6a51-e5a1-4bf9-b6f7-97b647aeb156.png)